app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -518,6 +518,14 @@ public void setCoordinates(@Nullable final LatLng coordinates) {
         this.coordinates = coordinates;
     }
 
+    /**
+     * Returns wikicode to use the media file on a MediaWiki site
+     * @return
+     */
+    public String getWikiCode() {
+        return String.format("[[File:%s|%s]]", filename, filename);
+    }
```

Comment:
The web provides an option to copy both the thumb and full image when it comes to using images in wikis. The corresponding Wiki codes suggested are:

- [[File:Title.jpg|thumb|Title]]
OR
- [[File:Title.jpg|Münster, St.-Paulus-Dom, Altarinsel -- 2019 -- 3815-9]]

Example:
![image](https://user-images.githubusercontent.com/12448084/83445606-e5603e80-a46a-11ea-87d1-547b71e6938e.png)
[Source file](https://commons.wikimedia.org/wiki/File:M%C3%BCnster,_St.-Paulus-Dom,_Altarinsel_--_2019_--_3815-9.jpg)

It would be nice if we could provide the users with both the wikicode and give them an option to choose from them. Alternatively, as we're concerned about usage of the image in a Wiki article, I think it makes sense to copy the wikicode for the "thumbnail" image rather than the wikicode for the full image.

Reply:
Thanks for the suggestion @sivaraam. cc: @misaochan to confirm which approach she prefers.

Reply:
> Alternatively, as we're concerned about usage of the image in a Wiki article, I think it makes sense to copy the wikicode for the "thumbnail" image rather than the wikicode for the full image.

I agree with the thumbnail approach, pictures are almost always displayed as thumbnails in wiki articles AFAIK.

Reply:
I have updated the code to use thumbnail.